media/js/firefox/new/experiment-privacy-dmt.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    // Check that cookies are supported.
+    if (typeof Mozilla.Cookies === 'undefined' || !Mozilla.Cookies.enabled()) {
+        return;
+    }
+
+    // Exclude existing Firefox users.
+    if (/\s(Firefox)/.test(navigator.userAgent)) {
+        return;
+    }
+
+    // courtesy of https://davidwalsh.name/query-string-javascript
+    function getUrlParam(name) {
+        name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+        var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
+        var results = regex.exec(location.search);
+        return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+    }
+
+    // if we already have a utm_content parameter, don't clobber existing stub attribution data.
+    if (!getUrlParam('utm_content')) {
```

Comment:
This experiment doesn't seem to be testing retention via stub attribution, so do we need this check?

Reply:
Hmm, could using `xv` here in the experiment config also conflict with existing xv params? For example, If I land on `/firefox/new/?xv=foo` and go into the experiment, I'm redirected to `?xv=foo&xv=priv-dmt&v=a`?

Maybe we should check for this param and exclude it from the experiment, instead of `utm_content`?

Reply:
Good catch. This was copy-pasta, I don't think we're worried about stub attribution here at all, we're only counting downloads.